package.json

```diff
@@ -183,9 +184,9 @@
     "raven": "^2.6.4",
     "raven-js": "^3.27.0",
     "rc-slider": "^7.0.8",
-    "react": "^16.5.0",
+    "react": "^16.8.2",
```

Comment:
Do we intend on upgrading `@types/react` in this PR too? When I was poking at this a few months back during the alpha release it seemed a bit messy, but that could have been resolved in some way.

Reply:
yeah, done! I forgot that people write typescript in force.